### ISSUE TITLE
Graphs are not bidirectional

### ISSUE THREAD

**Issue by RobDHess** [drorlab/atom3d#24]
Hi,

prot_df_to_graph and mol_df_to_graph result in graphs with connections that go only one way, which is due to the output of scipy's query_pairs. A possible fix would be to use:

edges = torch.cat((edges, edges.flip(dims=(0,))), dim=1)

**Comment by awfderry** [comment]
Hi, thanks for the catch! This should be fixed now.
